test/{{ repo-name }}_SUITE.erl

```diff
@@ -1,24 +1,29 @@
 -module({{ repo-name }}_SUITE).
 
--behavior(ct_suite).
+-behaviour(ct_suite).
 
 -include_lib("common_test/include/ct.hrl").
 -include_lib("stdlib/include/assert.hrl"). % Assertion macros for convenience
 
 -export([all/0, groups/0, init_per_testcase/2, end_per_testcase/2]).
 -export([success/1]).
 
+-spec all() -> list().
```

Comment:
Since you're adding specs for callbacks, you should match your specs with [the actuall callbacks](https://github.com/erlang/otp/blob/39c369a2c15bbd8de84afe2d7eb40d415020a498/lib/common_test/src/ct_suite.erl#L80-L118).
For instance in this case…
```suggestion
-spec all() -> [ct_suite:ct_test_def()].
```
…or…
```suggestion
-spec all() -> [ct_suite:ct_testname()].
```

Reply:
done